contents/docs/surveys/creating-surveys.mdx

```diff
@@ -159,3 +159,23 @@ Once you've set up your survey, click "Save as draft" to save your survey. This
 If you've created a [popover survey](#presentation), your survey will begin showing immediately to users matching your display conditions.
 
 If you've created an [API survey](#presentation), you'll need to add your [custom survey code](/docs/surveys/implementing-custom-surveys) before launching.
+
+## How PostHog handles partial survey responses
+
+You can enable partial responses in the "Completion conditions" section of the survey. This is enabled by default on new surveys. Notice that is requires at least the version `1.240.0` of `posthog-js`. It's currently only working for `posthog-js` library.
+
+**If partial responses are disabled (or if a user completes it in one go):**
+If partial responses are not a factor (e.g., the feature is disabled, or a user simply completes the survey fully in one interaction), PostHog sends a single `survey sent` event. This event is triggered upon full completion and submission of the survey and contains all the user's answers. The `$survey_completed` property in this event will be `true`.
+
+**When partial responses are enabled (default behavior):**
+If partial responses are enabled, PostHog tracks user progress as follows:
+
+1.  **Initial Response:** The first `survey sent` event is triggered as soon as the user answers at least one question. This initial event includes the answer(s) provided up to that point. The `$survey_completed` property in this event will be `false`.
+2.  **Subsequent Responses:** For each *additional* question the user answers, another `survey sent` event is captured. Each of these events will contain *all* responses submitted by the user for that survey instance up to that point, and `$survey_completed` will remain `false`.
```

Comment:
why do we include all answers again? feels like duplication on the client side
server side should merge the answers from multiple events

Reply:
[I created an issue for it here](https://github.com/PostHog/posthog/issues/32076) so we can track if it becomes a problem somewhere